packages/table/src/quadrants/tableQuadrant.tsx

```diff
@@ -99,15 +101,15 @@ export interface ITableQuadrantProps extends IProps {
      * A callback that renders either all of or just frozen sections of the table body.
      */
     renderBody: (
-        quadrantType: QuadrantType,
+        quadrantType?: QuadrantType,
         showFrozenRowsOnly?: boolean,
         showFrozenColumnsOnly?: boolean,
```

Comment:
oof this is messy. why are all the arguments optional? would be more accurate to declare the type as an override accepting either "3 defined params" or "0 params"

Reply:
Couldn't figure out how to do this syntactically. Leaving as is unless someone can show me how!

Reply:
...you could probably define both as types and then do typea | typeb? What's wrong with that? Other than ugly as all get out.

Reply:
Tried that; didn't compile.